pkg/util/loop/loop.go

```diff
@@ -160,35 +163,40 @@ func (loop *Device) AttachFromFile(image *os.File, mode int, number *int) error
 				return nil
 			}
 			syscall.Close(loopFd)
+			continue
 		} else {
```

Comment:
Once you add `continue` the `else` seems superfluous and confusing (and causes unnecessary indentation-only changes below).

Reply:
Fixed